extensions/MESA/MESA_EGL_image_cubemap.txt

```diff
@@ -0,0 +1,106 @@
+Name
+
+    MESA_EGL_image_cubemap
+
+Name Strings
+
+    GL_MESA_EGL_image_cubemap
+
+Contact
+
+    Simon Zeni, Status Holdings Ltd. <simon@zeni.ca>
+
+Contributors
+
+    Simon Zeni
+
+Status
+
+    Draft
+
+Version
+
+    Version 0.1, 2022/08/05
+
+Number
+
+    OpenGL ES Extension #546
+
+Dependencies
+
+    OpenGL ES 2.0 is required, or any previous version that offers support
+    for GL_ARB_texture_cube_map extension
+
+    GL_OES_EGL_image is required.
+
+    EGL 1.2 is required, with the EGL_KHR_image or EGL_KHR_image_base
+
+    This extension is written based on the wording of the OpenGL 2.0
+
+Overview
+
+    This extension provides a mechanism for creating cubemap textures from
+    EGLImage objects.
+
+    MESA_gl_texture_cubemap_image defines the process of creating an EGLImage
+    from an OpenGL ES cubemap texture.
+
+    The companion EGL_KHR_image_base and EGL_KHR_image extensions provide the
+    definition and rationale for EGLImage objects.
+
+    EGL extension specifications are located in the EGL Registry at
+            http://www.khronos.org/registry/egl/
+
+New Tokens
+
+    None.
+
+New Procedures and Functions
+
+    void EGLImageTargetCubemapMESA(enum target, eglImageOES image)
+
+Additions to Chapter 3 of the OpenGL 2.0 Specification (Rasterization)
+
+    In section 3.8.2 following the specification added by OES_EGL_image:
+
+    "It is also possible to specify cubemap textures images from existing
+    EGLImage objects. Images specified this way will be EGLImage sibilings
+    with the original EGLImage source and any other EGLImage targets.
+
+    The command
+
+        void EGLImageTargetCubemapMESA(enum target, eglImageOES image);
```

Comment:
Isn't this now a subset of https://registry.khronos.org/OpenGL/extensions/EXT/EXT_EGL_image_storage.txt - or am I missing something?

Reply:
I have somehow missed this extension, maybe because it's not implemented in mesa [1], thanks!
I'll take care of fixing that.

[1]: https://gitlab.freedesktop.org/mesa/mesa/-/blob/main/docs/features.txt#L403